angelsindustries/prototypes/overrides/components-bobs-entity-update.lua

```diff
@@ -28,177 +28,177 @@ if angelsmods.industries.components then
             ingredients = {
               {"!!"},
-              {"filter-inserter", 1},
-              {"circuit-orange-loaded", 2},
-              {"angels-servo-motor-3", 1},
-              {"titanium-bearing", 1},
-              {"angels-plating", 4},
+              {type = "item", name = "filter-inserter", amount = 1},
+              {type = "item", name = "circuit-orange-loaded", amount = 2},
+              {type = "item", name = "angels-servo-motor-3", amount = 1},
+              {type = "item", name = "angels-plating", amount = 4},
+              data.raw["titanium-bearing"] and {type = "item", name = "titanium-bearing", amount = 1} or nil
             }
           },
           {
             name = "turbo-stack-inserter", -- turbo stack inserter
             ingredients = {
               {"!!"},
-              {"stack-inserter", 1},
-              {"circuit-blue-loaded", 2},
-              {"angels-servo-motor-4", 3},
-              {"titanium-bearing", 1},
-              {"angels-plating", 8},
+              {type = "item", name = "stack-inserter", amount = 1},
+              {type = "item", name = "circuit-blue-loaded", amount = 2},
+              {type = "item", name = "angels-servo-motor-4", amount = 3},
+              {type = "item", name = "angels-plating", amount = 8},
+              data.raw["titanium-bearing"] and {type = "item", name = "titanium-bearing", amount = 1} or nil
             }
           },
           {
             name = "turbo-stack-filter-inserter", -- turbo stack filter inserter
             ingredients = {
               {"!!"},
-              {"stack-filter-inserter", 1},
-              {"circuit-blue-loaded", 3},
-              {"angels-servo-motor-4", 2},
-              {"titanium-bearing", 1},
-              {"angels-plating", 10},
+              {type = "item", name = "stack-filter-inserter", amount = 1},
+              {type = "item", name = "circuit-blue-loaded", amount = 3},
+              {type = "item", name = "angels-servo-motor-4", amount = 2},
+              {type = "item", name = "angels-plating", amount = 10},
+              data.raw["titanium-bearing"] and {type = "item", name = "titanium-bearing", amount = 1} or nil
             }
           },
           {
             name = "express-inserter", -- ultimate inserter
             ingredients = {
               {"!!"},
-              {"turbo-inserter", 1},
-              {"circuit-blue-loaded", 1},
-              {"angels-servo-motor-4", 2},
-              {"nitinol-bearing", 1},
-              {"angels-strut", 2},
+              {type = "item", name = "turbo-inserter", amount = 1},
+              {type = "item", name = "circuit-blue-loaded", amount = 1},
+              {type = "item", name = "angels-servo-motor-4", amount = 2},
+              {type = "item", name = "angels-strut", amount = 2},
+              data.raw["nitinol-bearing"] and {type = "item", name = "nitinol-bearing", amount = 1} or nil
```

Comment:
you can leave this check for nitinol bearing as our end goal is to not have these bob intermediates